shared/src/androidMain/kotlin/dev/johnoreilly/confetti/di/KoinAndroid.kt

```diff
@@ -23,11 +23,15 @@ import dev.johnoreilly.confetti.work.RefreshWorker
 import okhttp3.OkHttpClient
 import okhttp3.logging.LoggingEventListener
 import org.koin.android.ext.koin.androidContext
-import org.koin.androidx.workmanager.dsl.worker
+import org.koin.androidx.workmanager.dsl.workerOf
+import org.koin.core.module.dsl.bind
+import org.koin.core.module.dsl.new
+import org.koin.core.module.dsl.singleOf
+import org.koin.core.module.dsl.withOptions
 import org.koin.dsl.module
 
 actual fun platformModule() = module {
-    single<DateService> { AndroidDateService() }
+    singleOf(::AndroidDateService).withOptions { bind<DateService>() }
```

Comment:
`singleOf(::AndroidDateService).withOptions { bind<DateService>() }` can be written `singleOf(::AndroidDateService) { bind<DateService>() }`